src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+    private User user;
+
+    @BeforeEach
+     void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User();
+    }
+
+    @Test
+    void userIsNull_notOk() {
+        assertThrows(CustomException.class, () -> registrationService.register(null));
```

Comment:
The exception type 'CustomException' is not informative. It is better to use more specific exception names that indicate the nature of the error, such as 'RegistrationValidationException'.